src/components/GradeLevelFilter.jsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { Button, Popover } from '@material-ui/core';
+import ArrowDropDownIcon from '@material-ui/icons/ArrowDropDown';
+
+const useStyles = makeStyles((theme) => ({
+  buttonStyle: {
+    margin: theme.spacing(1),
+    height: theme.spacing(4),
+  },
+  popOverWindow: {
+    minWidth: '200px',
+  },
+}));
+
+const gradeList = {
+  12: {
+    value: 'Highschool Senior',
+    isChecked: false,
+  },
+  13: {
+    value: 'College Freshman',
+    isChecked: false,
+  },
+  14: {
+    value: 'College Sophomore',
+    isChecked: false,
+  },
+  15: {
+    value: 'College Junior',
+    isChecked: false,
+  },
+  16: {
+    value: 'College Senior',
+    isChecked: false,
+  },
+};
```

Comment:
Instead of this you could map the string to a boolean. E.g.

```js
const gradeList = {
  'Highschool Senior': false,
  // etc.
};
```

Or even better, since the values are just booleans, you could use a [Set](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set/Set) to track checked options.

```js
const selected = useState(Set());

// use selected.add() and selected.remove() on selecting/deselecting a checkbox.
```

Reply:
Although I think I see now maybe why this was done. Maybe in anticipation of URL parameters where encoding numbers might make more sense. That being the case, I think this PR can stay focused on just the styling if you prefer. Maybe we should discuss with the others what might make sense to encode in the URL for grade level selections.

Reply:
Ok I can stay focused on the styling and then once we figure out how to encode the parameters for grades I can go ahead and figure out how to do this using a `Set`. Basically as a box is checked the onClick function would handle the adding of the grade selected and when the grade is already selected it would just search the set if found then just remove it, there would be no need for boolean, correct?

Reply:
> once we figure out how to encode the parameters for grades I can go ahead and figure out how to do this using a Set

Not what I meant. I meant the map style you have could be necessary when encoding the selections via URL parameters. For styling/prototyping purposes a set is perfectly fine.

> Basically as a box is checked the onClick function would handle the adding of the grade selected and when the grade is already selected it would just search the set if found then just remove it, there would be no need for boolean, correct?

Right, clicking just toggles the presence of the selection.